test/manual/amp-sidebar-1.0.amp.html

```diff
@@ -52,6 +52,11 @@
   margin-right: auto;
 }
 
+.toolbar-compensation {
```

Comment:
this should be` [toolbar] + .toolbar-compensation  ` so that it gets applied only when the toolbar is visible - keep in mind that developers could copy your approach from here :)